pkg/entities/nfts.go

```diff
@@ -654,6 +655,41 @@ func (e *Entity) GetAllNFTSalesTracker() ([]response.NFTSalesTrackerResponse, er
 	return resp, nil
 }
 
+func (e *Entity) DeleteNFTSalesTracker(guildID, contractAddress string) error {
+	salesTracker, err := e.repo.NFTSalesTracker.GetNFTSalesTrackerByContractAndGuildID(guildID, contractAddress)
```

Comment:
can we use DeleteNFTSalesTrackerByAddress

Reply:
query first to get ID to delete because the request doesn't include ID

Reply:
can query by address instead of id ? will use 1 step only